iree/compiler/Conversion/Passes.h

```diff
@@ -0,0 +1,53 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef IREE_COMPILER_CONVERSION_PASSES_H_
+#define IREE_COMPILER_CONVERSION_PASSES_H_
+
+#include <memory>
+
+#include "mlir/Pass/Pass.h"
+
+namespace mlir {
+namespace iree_compiler {
+
+// Registers all conversion passes in this directory.
+void registerConversionPasses();
+
+//------------------------------------------------------------------------------
+// Conversions into Linalg
+//------------------------------------------------------------------------------
+
+/// Creates a pass to fuse operations on tensors.
```

Comment:
Nit: it's better to be clear that this is "fuse Linalg operations on tensors".

Reply:
Actually, leaving as is since this changes the way the pass is defined too and I think adds opacity (and is different from how everything else does it).